### PR TITLE
Publish beta package

### PR DESCRIPTION
**Pull request by pascalw**
(Note this is based on https://github.com/Wakamai-Fondue/wakamai-fondue-cli/pull/5, I'll rebase this PR when that is merged).

This PR makes a few changes to package.json in preparation for publication npm:

* Package name is changed to `@wakamai-fondue/cli`
* Keywords are added
* README is included in the package so it can be displayed on npmjs.com

See published (beta) package here: https://www.npmjs.com/package/@wakamai-fondue/cli

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,13 @@
 		"wakamai-fondue": "./cli.js"
 	},
 	"files": [
-		"cli.js"
+		"cli.js",
+		"README.md"
+	],
+	"keywords": [
```
Comment (pascalw) [inline package.json:16]: @RoelN any other keywords we should add?

**Comment by pascalw** [comment]
I noticed NPM displays https://www.npmjs.com/package/fondue as a dependency [1], because the WF engine package name is now `fondue`. I'll rename the engine to `@wakamai-fondue/engine`.

Screenshot 2020-09-25 at 08 44 44

File: package.json (hunk 1 above)
Comment (RoelN) [reply package.json:16]: I think these would be nice to add:

- variable fonts
- truetype
- otf
- ttf
- woff
- woff2
- css

**Review by RoelN — APPROVED** [review]
Nice, thanks @pascalw!

File: package.json (hunk 1 above)
Comment (pascalw) [reply package.json:16]: Added!

**Comment by RoelN** [comment]
:greencheckmark: !

**Pull request merged**